.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,14 @@ resolves #1235
 ### This pullrequest changes
 
 <!-- Please describe what your pullrequest is changing -->
+
+---
+### Pull Request Readiness Checklist
+
+- [ ] I agree to contribute to the project under OpenCV ([BSD](https://github.com/opencv/opencv/blob/master/LICENSE)) license
+- [ ] The PR is proposed to proper branch ([details](https://github.com/opencv/opencv/wiki/How_to_contribute#before-you-start-contributing-you-should))
+- [ ] There is reference to original bug report and related work
+- [ ] There is PR with opencv_extra changes, if any
+- [ ] There is test, if code has been changed
```

Comment:
> PR with opencv_extra changes
> test

I believe `test` item should go first. `opencv_extra` should be mentioned as a place for new `testdata` (perhaps in the same item)

```
- [ ] There is test (required if code has been changed).
- [ ] Updated testdata (if needed): https://github.com/opencv/opencv_extra/pull/000
```